### ISSUE TITLE
[WIP] Rails 5: Rack 2 breaks capybara's click_button method

### ISSUE THREAD

**Issue by jasnow** [jnicklas/capybara#1592]
Appears that rack 2 (for Rails 5) has broken capybara's click_button method.
Now getting "undefined method `normalize_params' for Rack::Utils:Module"
error when we using capybara's click_button method.

 * After porting to Rails 5, I get this error with at least 3 repos:
```
   Failure/Error: click_button "Create Task"
     NoMethodError:
       undefined method `normalize_params' for Rack::Utils:Module
```
To Reproduce, you can do the following:
  * Example repo (I remove all but 1 test) with error:
    * git clone https://github.com/jasnow/tasks5lite.git
    * After set up, run "rake" and you will get this error:
```
        Failure/Error: click_button "Create Task"
        NoMethodError:
          undefined method `normalize_params' for Rack::Utils:Module
```
  * After tracking down the trigger line, I found this line
    * Rack::Utils.normalize_params(params, key, value)

    in this method:
    * merge_param!(params, key, value)

    in this file:
    * capybara-2.5.0/lib/capybara/rack_test/form.rb

Also found these references interesting:
  * http://apidock.com/rails/Rack/Utils/normalize_params
    * Where I found this sentence: 
      * This method is deprecated or moved on the latest stable version. The last existing version (v2.3.2) is shown here.

  * http://openwall.com/lists/oss-security/2015/06/16/14
    * Mentioned security-related changes to normailze_params inside rack.

  * https://thewebfellas.com/blog/rails-2-3-8-rack-1-1-and-the-curious-case-of-the-missing-quotes
    * Unclear if this is helpful.

**Comment by twalpole** [comment]
This should already be fixed in the master branch, please try using that

**Comment by jasnow** [comment]
You are right - Sorry for the issue.

**Comment by jasnow** [comment]
When is the next release of capybara gem?

**Comment by twalpole** [comment]
No current dare set - use the master branch

**Comment by jasnow** [comment]
OK

**Comment by embs** [comment]
I've faced this same issue and upgrading to a newer version fixed it for me. In my case, I've used:

```
gem 'capybara', '~> 2.8'
```

**Comment by chemturion** [comment]
I just upgraded from Rails 4.2.7 to 5.0.0, and had to set `capybara-webkit` 1.8.0, and then `bundle update capybara`, which set capybara to 2.6.2 and solved the problem for me.

**Comment by twalpole** [comment]
@chemturion Capybara 2.6.2 is obsolete (released Jan 2016) as is capybara-webkit 1.8 (Jan 2016) - You want to be running newer than that.
